pkgs/top-level/python2-packages.nix

```diff
@@ -524,10 +524,12 @@ with self; with super; {
 
   scandir = callPackage ../development/python-modules/scandir { };
 
-  scikitlearn = callPackage ../development/python-modules/scikitlearn/0.20.nix {
+  scikit-learn = callPackage ../development/python-modules/scikit-learn/0.20.nix {
     inherit (pkgs) gfortran glibcLocales;
   };
 
+  scikitlearn = self.scikit-learn;
```

Comment:
I think this shouldn't be necessary.

Reply:
Why not? Won’t changing the name without adding an alias potentially break out-of-tree user nix code, flakes, etc?

Reply:
@DavHau I didn't notice you removed it from `python-packages.nix` as well. I just meant the line in `python2-packages.nix` shouldn't be necessary.

@rmcgibbo Any alias in `python-packages.nix` will be available for both Python 2 and Python 3.